### PR TITLE
Release process

### PR DESCRIPTION
**Pull request by Thomasvdh92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,39 @@ Exporting and importing a sub-tree, or a single page is supported.
...
+Checklist:
+
+- Update CONTRIBUTORS.rst if necessary
+- Update CHANGELOG.txt
+- Everything is commited, clean checkout
+- Run the tests
+- Create a tag
+- Push said tag
+- Generate distribution packages for this package
+- Upload
```
Comment (allcaps) [inline README.md]: Can't these be dropped? As these steps are specified below.

File: docs/index.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Exporting and importing a sub-tree, or a single page is supported.
 
 - Related objects (such as images, documents, and snippets) are not translated or duplicated.
 - Pages will be in draft mode. Pages need to be reviewed and published.
+- Nested streamfield are known to cause issues since their database values are not correct. The page will be created but causes errors.
```
Comment (allcaps) [inline docs/index.md]: We call them nested streamfields, but they are `StreamBlock`s right?

And do they always fail? Or after a certain level of nesting?

Where do these errors occur? `causes errors` -> `cause errors on upload` or something.

File: README.md (hunk 1 above)
Comment (allcaps) [reply README.md]: Note, 27-31

File: docs/index.md (hunk 2 above)
Comment (Thomasvdh92) [reply docs/index.md]: StreamBlocks indeed, thanks for pointing out ;)

**Review by allcaps — APPROVED** [review]

**Pull request merged**